Code/GraphMol/Wrap/SubstanceGroup.cpp

```diff
@@ -105,11 +177,20 @@ struct sgroup_wrap {
              "SubstanceGroup.\n"
              " n.b. some properties cannot be converted to python types.\n");
     python::def("GetMolSubstanceGroups", &getMolSubstanceGroups,
-                "returns the SubstanceGroups for a molecule (if any)",
+                "returns a copy of the molecule's SubstanceGroups (if any)",
                 python::with_custodian_and_ward_postcall<0, 1>());
+    python::def("GetMolSubstanceGroupWithIdx", &getMolSubstanceGroupWithIdx,
+                "returns a particular SubstanceGroup from the molecule",
+                python::return_internal_reference<
+                    1, python::with_custodian_and_ward_postcall<0, 1>>());
     python::def("ClearMolSubstanceGroups", &clearMolSubstanceGroups,
                 "removes all SubstanceGroups from a molecule (if any)");
-    // FIX: needs something tying the lifetime to the mol
+    python::def("CreateMolSubstanceGroup", &createMolSubstanceGroup,

```

Comment:
This has no corollary in C++ land, correct?  Perhaps there should be one.

Reply:
C++ doesn't really need an equivalent. The code there is:
```
  SubstanceGroup sg(&mol, type);
  addSubstanceGroup(mol, sg);
```